billboardd/daemon.h

```diff
@@ -295,6 +303,13 @@ class Daemon : public QObject {
             context->subscribeTo(interestingProperties);
             gconfmon->subscribeTo(interestingProperties);
 
+            if (interestingProperties.contains("tklock")) {
+                QObject::connect(&tklock, SIGNAL(locked()),
```

Comment:
Good point, but e.g. when showing the preview in the Billboard app, then using the `{tklock}` property should probably show the right value, no? If we don't want to show the value, we probably shouldn't allow using it as property, but just emit the `contextChanged()` signal, which has the same effect (updating the thing).

As you said on TMO IIRC, there's no battery drainage if it updates every time the screen is locked, so we can probably do it unconditionally on `locked()`.